source/codegen/metadata/nixnet/functions.py

```diff
@@ -1025,6 +1037,14 @@
             {
                 'direction': 'out',
                 'name': 'numberOfBytesReturned',
+                'type': 'u32',
+                'include_in_proto': False
+            },
+            {
+                'direction': 'in',
```

Comment:
Are you working on adding these to scrapigen?

Reply:
Yes, all of the changes from here will be backported to scrapigen.